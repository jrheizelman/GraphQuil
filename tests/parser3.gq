int main()
{
	int a;
	String b;
	while(true) {
		int h;
		a = 1;
		b = "gemma";
		print(a);
		print(b);
	}
}