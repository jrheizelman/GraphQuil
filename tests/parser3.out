int main()
{
a int;
b String;
while (true) {
{
a = 1;
b = "gemma";
print(a);
print(b);
}
}}